docs/reference/ml/delayed-data-detection.asciidoc

```diff
@@ -0,0 +1,41 @@
+[role="xpack"]
+[[ml-delayed-data-detection]]
+=== Working with delayed data
+
+In short, delayed data are any documents that are indexed *after* a datafeed has already gathered data for the 
+point in time referenced in the data time-field value.
+
+When you create a datafeed, you can specify a {ref}/ml-datafeed-resource.html[`query_delay`] setting.
+This setting enables the datafeed to wait for some time past real-time, which means any "late" data in this period
+is fully index before the datafeed tries to gather it. However, if the setting is set too low, the datafeed may query
+for data before it has been indexed and consequently miss that document. Conversely, if it is set too high,
+analysis drifts farther away from real-time. The balance that is struck depends upon each use case and
+the environmental factors of the cluster.
+
+==== Why worry about delayed data?
+
+This is a particular prescient question. If there are data missing randomly (perhaps due to ingest delays), the results of
+certain types of functions are not really affected. Statistically, it all comes out ok in the end as the missing data is
+distributed randomly. An example would be a `mean` metric for a field in a large collection of data.
```

Comment:
Is there a difference between "missing data" and "delayed data"?  I would think the former never arrives, whereas the latter arrives late.

Reply:
Correct, but the concepts here expand out to the more general case of "missing data". I can restrict it down to just "late" for clarity as it is still correct.

Reply:
Thanks, I think either this page needs to be restricted to talking about "delayed data" or we need to add more information about missing and delayed data differ (and how/if they're handled differently).  The latter sounds interesting to me, but maybe beyond the scope of this PR.